File: C_Sharp_code/Example/WindowsFormsApp/VB_demo06/Form1.vb
Public Class Form1
    Public Structure Student
        Public no As Integer
        Public name As String
    End Structure

    Private Sub button1_Click(sender As Object, e As EventArgs) Handles button1.Click
        Dim s As Student
        s.no = 49780048
        s.name = "KEN"
        Console.WriteLine("no:{0}, name:{1}", s.no, s.name)
    End Sub
End Class

File: C_Sharp_code/Example/WindowsFormsApp/CS_demo06/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CS_demo06
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        struct Student
        {
            public int no;
            public string name;
        }
        private void button1_Click(object sender, EventArgs e)
        {
            Student s = new Student();
            s.no = 49780048;
            s.name = "Ken";
            Console.WriteLine("no:{0}\r\nname:{1}", s.no, s.name);
        }
    }
}
